### DISCUSSION TITLE
Welcome to Conan-Shop Discussions!

### DISCUSSION THREAD

**Discussion by irrelevantgamers** [irrelevantgamers/Conan-Shop#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adrien-pago** [comment]
Bonjour,

J'ai essayé de reprendre ton bot discord pour le Conan Shop et le mettre sur mon serveur.
Mais je n'arrive pas à lui donner le bon chemin pour la base de donnéesLocation et Loglocation peux tu m'aider ? Je loue un serveur Conan avec Gportal

Bonjour,

J'ai essayé de reprendre votre bot discord pour la boutique Conan et de le mettre sur mon serveur.
Mais je ne peux pas lui donner le bon chemin pour la databaseLocation et Loglocation pouvez-vous m'aider ? Je loue un serveur Conan avec gportal
